### ISSUE TITLE
[TF2.0] Bug when export BatchNormalization layer

### ISSUE THREAD

**Issue by qixiuai** [tensorflow/tensorflow#30471]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): 
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
   Linux Ubuntu 18.04
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow installed from (source or binary):
  binary
- TensorFlow version (use command below):
  2.0.0-beta0
- Python version:
  python 3.6.8
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version:
  cuda 10, 
- GPU model and memory:
  GTX 1080ti

**Describe the current behavior**
```
W0708 11:01:14.168962 140059689457472 saved_model.py:721] Skipping full serialization of 
object <tensorflow.python.keras.layers.normalization_v2.BatchNormalization object at 
0x7f61c004a978>, because an error occurred while tracing layer functions. Error message: 
Expected Operation, Variable, or Tensor, got None
```
**Describe the expected behavior**
There should be no error message.

**Code to reproduce the issue**
```
import tensorflow as tf
from tensorflow.keras.layers import BatchNormalization

class BNModel(tf.Module):

    def __init__(self):
        super(BNModel, self).__init__()
        self.bn = BatchNormalization()

    @tf.function(input_signature=[tf.TensorSpec([None, 2], tf.float32)])
    def __call__(self, x):
        return self.bn(x)

if __name__ == '__main__':
    module = BNModel()
    import numpy as np
    arr = np.random.randn(2,2)
    module(arr)                                                                                                                                           
    signatures = {"serving_default": module.__call__}                                                                         
    tf.saved_model.save(module, "saved_models", signatures=signatures)                                                      

```

**Other info / logs**
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

**Comment by gadagashwini** [comment]
Was able to reproduce the reported issue with Tensorflow 2.0.0.beta0 and It works as expected with Tenosrflow 2.0.0.beta1. Thanks!

**Comment by qixiuai** [comment]
My problem is solved. Thank you very much.  @gadagashwini
